index.html

```diff
@@ -6348,6 +6348,41 @@ <h2>Acknowledgements</h2>
 standards process.
     </p>
 
+    <p>
+The Editors gratefully acknowledge the work that led to the creation of this
+specification, and extend a sincere appreciation to those individuals that
+worked on technologies and specifications that deeply influenced this Working
+Group. Namely, the work of Phil Zimmerman, Jon Callas, Lutz Donnerhacke, Hal
+Finney, David Shaw, and Rodney Thayer on <a
+href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy"> Pretty Good Privacy
+(PGP)</a> in the 1990s and 2000s.
+    </p>
+
+    <p>
+In the mid-2010s, preliminary implementations of what would become Decentralized
+Identifiers were <a
+href="https://web-payments.org/minutes/2014-05-07/#topic-1">built</a> in
+collaboration with Jeremie Miller's Telehash project and the W3C Web Payments
+Community Group's work led by Dave Longley and Manu Sporny.
+Some of the <a
```

Comment:
Suggestion to also mention XDI.org. My recollection is that members of the W3C Web Payments Community Group and members of the XDI.org Registry Working Group came together to do the initial work at RWoT.

```suggestion
Community Group's work led by Dave Longley and Manu Sporny.
Around the same time, the XDI.org Registry Working Group <a href="https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/">began exploring</a> decentralized
technologies for replacing its existing identifier registry.
Some of the <a
```

Reply:
JFYI, the XDI.org work happened (I believe) ~1.5 years later, so we should not say "around the same time".

Reply:
I think the XDI.org work started in mid 2015, yes that's later than the Web Payments work, but if the temporal context is "the mid-2010s", then I'd say it's still "around the same time". But we can also change it to "soon after" or something similar, do you have a suggestion?

Reply:
"Around the same time" => "Around a year later" sounds like it would work.

Reply:
is `https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/` the best link?

Reply:
> "Around the same time" => "Around a year later" sounds like it would work.

Changed that.

Reply:
> is `https://docs.google.com/document/d/1EP-KhH60y-nl4xkEzoeSf3DjmjLomfboF4p2umF51FA/` the best link?

Yes, this was a series of meetings over a few months where all the work was done via Google Docs..